### PR TITLE
Change account stats mysql statements to add new container stats to db

### PR DESCRIPTION
**Pull request by justinlin-linkedin**
Changing the sql statement to add physical storage usage and number of blobs to mysql database. This is the same as this PR https://github.com/linkedin/ambry/pull/1943

### CODE REVIEW

**Review by cgtz — APPROVED** [review]

**Review by SophieGuo410 — APPROVED** [review]
lgtm.

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java [hunk 1]
```diff
@@ -80,6 +85,9 @@
    */
   void updateStorageUsage(String clusterName, String hostname, int partitionId, short accountId, short containerId,
       long storageUsage) throws SQLException {
+    // TODO: adding real physical storage usage and number of blobs here
+    long physicalStorageUsage = storageUsage;
+    long numberOfBlobs = 1L;
     try (Connection connection = dataSource.getConnection()) {
       try (PreparedStatement insertStatement = connection.prepareStatement(insertSql)) {
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java:92]: Do we need to escape the sql statement to guard ourselves against the sql injections? This applies to all other sql statements in this file.

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java [hunk 2]
```diff
@@ -80,6 +85,9 @@
    */
   void updateStorageUsage(String clusterName, String hostname, int partitionId, short accountId, short containerId,
       long storageUsage) throws SQLException {
+    // TODO: adding real physical storage usage and number of blobs here
+    long physicalStorageUsage = storageUsage;
+    long numberOfBlobs = 1L;
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java:90]: Nit: final

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java [hunk 3]
```diff
@@ -124,8 +137,12 @@ void queryStorageUsageForHost(String clusterName, String hostname, ContainerUsag
             int accountId = resultSet.getInt(ACCOUNT_ID_COLUMN);
             int containerId = resultSet.getInt(CONTAINER_ID_COLUMN);
             long storageUsage = resultSet.getLong(STORAGE_USAGE_COLUMN);
+            long physicalStorageUsage = resultSet.getLong(PHYSICAL_STORAGE_USAGE_COLUMN);
+            long numberOfBlobs = resultSet.getLong(NUMBER_OF_BLOBS_COLUMN);
             long updatedAtMs = resultSet.getTimestamp(UPDATED_AT_COLUMN).getTime();
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java:142]: Nit: final

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java [hunk 4]
```diff
@@ -252,14 +269,21 @@ public StorageBatchUpdater(int maxBatchSize) throws SQLException {
      */
     public void addUpdateToBatch(String clusterName, String hostname, int partitionId, short accountId,
         short containerId, long storageUsage) throws SQLException {
+      // TODO: adding real physical storage usage and number of blobs here
+      long physicalStorageUsage = storageUsage;
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java:273]: nit: final

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AggregatedAccountReportsDao.java [hunk 5]
```diff
@@ -102,6 +110,9 @@
    */
   void updateStorageUsage(String clusterName, short accountId, short containerId, long storageUsage)
       throws SQLException {
+    // TODO: adding real physical storage usage and number of blobs here
+    long physicalStorageUsage = storageUsage;
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/AggregatedAccountReportsDao.java:114]: nit: final

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AggregatedAccountReportsDao.java [hunk 6]
```diff
@@ -102,6 +110,9 @@
...
+    long numberOfBlobs = 1L;
     try (Connection connection = dataSource.getConnection()) {
       try (PreparedStatement insertStatement = connection.prepareStatement(insertSql)) {
         long startTimeMs = System.currentTimeMillis();
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/AggregatedAccountReportsDao.java:118]: nit: final

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AggregatedAccountReportsDao.java [hunk 7]
```diff
@@ -191,7 +207,10 @@ private void queryContainerUsageForClusterInternal(boolean forMonthly, String cl
             int accountId = resultSet.getInt(ACCOUNT_ID_COLUMN);
             int containerId = resultSet.getInt(CONTAINER_ID_COLUMN);
             long storageUsage = resultSet.getLong(STORAGE_USAGE_COLUMN);
-            func.apply((short) accountId, (short) containerId, storageUsage);
+            long physicalStorageUsage = resultSet.getLong(PHYSICAL_STORAGE_USAGE_COLUMN);
+            long numberOfBlobs = resultSet.getLong(NUMBER_OF_BLOBS_COLUMN);
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/AggregatedAccountReportsDao.java:211]: nit: final

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AggregatedAccountReportsDao.java [hunk 8]
```diff
@@ -327,12 +346,19 @@ public AggregatedStorageBatchUpdater(int maxBatchSize) throws SQLException {
      */
     public void addUpdateToBatch(String clusterName, short accountId, short containerId, long storageUsage)
         throws SQLException {
+      // TODO: adding real physical storage usage and number of blobs here
+      long physicalStorageUsage = storageUsage;
+      long numberOfBlobs = 1L;
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/AggregatedAccountReportsDao.java:351]: nit: final

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/PartitionClassReportsDao.java [hunk 9]
```diff
@@ -307,8 +312,11 @@ void queryAggregatedPartitionClassReport(String clusterName, PartitionClassConta
             int accountId = rs.getInt(ACCOUNT_ID_COLUMN);
             int containerId = rs.getInt(CONTAINER_ID_COLUMN);
             long usage = rs.getLong(STORAGE_USAGE_COLUMN);
+            long physicalStorageUsage = rs.getLong(PHYSICAL_STORAGE_USAGE_COLUMN);
+            long numberOfBlobs = rs.getLong(NUMBER_OF_BLOBS_COLUMN);
             long updatedAt = rs.getTimestamp(UPDATED_AT_COLUMN).getTime();
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/PartitionClassReportsDao.java:317]: nit: final

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/PartitionClassReportsDao.java [hunk 10]
```diff
@@ -376,13 +384,20 @@ public StorageBatchUpdater(int maxBatchSize) throws SQLException {
      */
     public void addUpdateToBatch(String clusterName, String partitionClassName, short accountId, short containerId,
         long usage) throws SQLException {
+      // TODO: adding real physical storage usage and number of blobs here
+      long physicalStorageUsage = usage;
+      long numberOfBlobs = 1;
```
Comment (tbingol) [inline ambry-mysql/src/main/java/com/github/ambry/accountstats/PartitionClassReportsDao.java:389]: nit: final

**Review by tbingol — CHANGES_REQUESTED** [review]

File: ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java (hunk 1 above)
Comment (justinlin-linkedin) [reply ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java:92]: I thought prepared statement would differentiate the sql code from the user input, thus prevent sql injections.

**Review by tbingol — APPROVED** [review]
LGTM

**Pull request merged**